_posts/2019-10-30-cockpit-206.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Cockpit 206
+author: Gundersanne
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-206
+category: release
+summary: Cockpit with machines, firewall, and login page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Machines: NIC deletion
+
+NICs can now be removed from virtual machines.
+
+![Machines NIC deletion](/images/machines-nic-deletion.png)
+
+### login: Check reuse password by default
+
+Cockpit is heavily restricted without sudo rights, so make reusing the password
+for privileged tasks the default.
+
+![login reuse password checkbox is checked](/images/login-reuse-password-checked.png)
+
+### Firewall: Removing services or zones are now at least 2 clicks
```

Comment:
It feels like the implementation should move into the detailled text, and this should just describe the effect. Perhaps "Prevent accidental deletion"?